### PR TITLE
Convert app to use Compose file 2

### PR DESCRIPTION
**Pull request by bfirsh**
If looks good I will merge on release of Compose 1.6.

/cc @aanand @mgoelzer

### CODE REVIEW

**Comment by aanand** [comment]
Nice!

The next Compose RC will refuse to run this because the `myvolume` volume isn't explicitly declared. Add this to the file:

```yaml
volumes:
  myvolume:
```

**Comment by bfirsh** [comment]
@aanand thanks! how come that is required?

**Comment by aanand** [comment]
For one thing, mistyped volume names would otherwise result in the silent creation of a volume with the wrong name, which would lead to some pretty unpleasant behaviour in apps where named volumes are shared.

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+version: 2
```
Comment (dnephin) [inline docker-compose.yml]: This needs to be `version: "2"`

the version is a string

Comment (bfirsh) [inline docker-compose.yml]: thanks!

**Pull request merged**